apps/autoscheduler/DefaultCostModel.cpp

```diff
@@ -176,18 +176,18 @@ class DefaultCostModel : public CostModel {
             *(cost_ptrs(i)) = dst(i);
             if (std::isnan(dst(i))) {
                 any_nans = true;
```

Comment:
This is all an elaborate error message for when the network predicts nan, and should be printed unconditionally

Reply:
bump

Reply:
PTAL